packages/build-essentials/src/webpack.config.js

```diff
@@ -33,8 +33,8 @@ const webpackConfig = {
                 loader: 'json-loader'
             },
             {
-                test: /\.(woff|woff2|eot|svg)$/,
-                loader: 'url-loader?limit=100000'
+                test: /\.(woff|woff2)$/,
```

Comment:
I think this loader can be dropped, since it is removed from the package.json.

Any reasons, why it's still here?